code/modules/virus2/effect.dm

```diff
@@ -392,10 +392,12 @@
 
 /datum/disease2/effect/spawn/activate(var/mob/living/carbon/mob,var/multiplier)
 	//var/mob/living/carbon/human/H = mob
-	var/placemob = locate(mob.x + pick(1,-1), mob.y, mob.z)
+	var/turf/T = locate(mob.x + pick(1,-1), mob.y, mob.z)
```

Comment:
This is kind of a weird pattern, the roach always goes to your left or to your right, it'd make more sense if it went in front of you or just always in your tile. But hey this is just a bugfix.